doc/examples/tiled/lib/main.dart

```diff
@@ -1,14 +1,43 @@
 import 'package:flame/components/tiled_component.dart';
 import 'package:flame/game.dart';
-import 'package:flutter/widgets.dart';
+import 'package:flame/flame.dart';
+import 'package:flame/position.dart';
+import 'package:flame/animation.dart';
+import 'package:flutter/widgets.dart' hide Animation;
+import 'package:tiled/tiled.dart' show ObjectGroup, TmxObject;
 
 void main() {
+  Flame.images.load('coins.png');
   final TiledGame game = TiledGame();
   runApp(game.widget);
 }
 
 class TiledGame extends BaseGame {
+  TiledComponent tiledMap;
+  Animation a;
```

Comment:
Since you are using `BaseGame` here, you could use an instance of `AnimationComponent` for each object instead of manual drawing the coins on the game.

Reply:
Huummm.. I didnt get that, but I will see how it done.